src/component/admin/groups/GroupsList/GroupCard/GroupCard.tsx

```diff
@@ -92,13 +112,26 @@ export const GroupCard = ({ group }: IGroupCardProps) => {
                             <ConditionallyRender
                                 condition={group.projects.length > 0}
                                 show={group.projects.map(project => (
-                                    <Badge
-                                        color="secondary"
-                                        icon={<TopicOutlinedIcon />}
-                                        sx={{ marginRight: 0.5 }}
+                                    <Tooltip
+                                        key={group.id}
```

Comment:
One group might have multiple projects, so I think we should use a project-bound identifier instead here.
```suggestion
                                        key={project}
```

Reply:
Ah seems to be typo. Nice catch.